Package.swift

```diff
@@ -1,4 +1,4 @@
-// swift-tools-version: 5.7
+// swift-tools-version: 5.9
```

Comment:
This is an unfortunate change, do we really need it?

Reply:
ups, reverted.